emark/registry.py

```diff
@@ -0,0 +1,32 @@
+from django.urls import reverse_lazy
+
+
+class EmarkRegistry:
```

Comment:
This sure is a very sophisticated dictionary ;)

Reply:
Yeah, not sure how I am improving here.
But would be happy to learn, if you provide a suggestion 👍 👍

Reply:
Tried to simplify this in the latest commit.